### ISSUE TITLE
E: The repository 'https://xpra.org/beta bookworm Release' is not signed.

### ISSUE THREAD

**Issue by SkyperTHC** [Xpra-org/xpra#3861]
Hello,

Tested on kalilinux/kali-last-release. 
Uhm. Why isnt this working anymore? It errors that the release file is not signed.

```shell
bash -c '{ `### only Beta has no conflict with python 3.11.3 and libprocps8` \
                && wget -O "/usr/share/keyrings/xpra-2022.gpg" https://xpra.org/xpra-2022.gpg \
                && wget -O "/etc/apt/sources.list.d/xpra-beta.list" https://xpra.org/repos/bookworm/xpra-beta.list \
                && apt-get update \
                && apt-get install -y --no-install-recommends "xpra" "xpra-html5" "xpra-x11"; }'
```

Result:
```
E: The repository 'https://xpra.org/beta bookworm Release' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by totaam** [comment]
First, bookworm no longer requires the beta repository AFAIK.
Second, as per https://lists.devloop.org.uk/pipermail/shifter-users/2023-May/002974.html, please see #3846

**Comment by SkyperTHC** [comment]
Hmm. The xpra-beta.list and also xpra.list both *demand* xpra-2022.gpg:

xpra-beta.list:
```
deb [arch=amd64,arm64 signed-by=/usr/share/keyrings/xpra-2022.gpg] https://xpra.org/beta/ bookworm main
```

xpra.list:
```
deb [arch=amd64,arm64 signed-by=/usr/share/keyrings/xpra-2022.gpg] https://xpra.org/ bookworm main
```

The key from `xpra.org/gpg.asc` or `xpra.org/xpra.asc` wont work (the rep is signed with a different key; the Key ID is different - renaming the key wont do it).

Same error happens with the install instructions from https://github.com/Xpra-org/xpra/wiki/Download#-linux:
```
DISTRO=bookworm
wget -O "/usr/share/keyrings/xpra.asc" https://xpra.org/xpra.asc
wget -O "/etc/apt/sources.list.d/xpra.sources" https://xpra.org/repos/$DISTRO/xpra.sources
apt update
```

```
Err:2 https://xpra.org bookworm InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 18ADB31CF18AD6BB
```

**Comment by SkyperTHC** [comment]
Is there any workaround to install xpra in the meantime?


That's a cut & paste from xpra.org's webpage how to install and it fails:

```
docker run --rm -it  kalilinux/kali-last-release
apt-get update -y
apt-get install -y wget ca-certificates
```

Cut & Paste from xpra.org:

```
wget -O "/usr/share/keyrings/xpra.asc" https://xpra.org/xpra.asc
wget -O "/etc/apt/sources.list.d/xpra.sources" https://raw.githubusercontent.com/Xpra-org/xpra/master/packaging/repos/bookworm/xpra.sources
apt-get update
```

results in:
```
Err:1 https://xpra.org bookworm InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 18ADB31CF18AD6BB
```
